presentation/matrix_view.py

```diff
@@ -203,53 +208,71 @@ def get_thresholds(self, metric: MetricType) -> Thresholds:
         return self._config.packet_loss
 
     @staticmethod
-    def get_metric_value(metric: MetricType, cell: MeshColumn) -> MetricValue:
+    def get_metric_value(metric: MetricType, health: HealthItem) -> MetricValue:
```

Comment:
What about to add  `get_value(metric: MetricType)` method to the `HealthItem` class and get rid of this one static method?